sycl/test/devicelib/cmath_fp64_test.cpp

```diff
@@ -1,9 +1,8 @@
-// UNSUPPORTED: windows
-// RUN: %clangxx -fsycl -c %s -o %t.o
-// RUN: %clangxx -fsycl %t.o %sycl_libs_dir/libsycl-cmath-fp64.o -o %t.out
+// REQUIRES: cpu, linux
+// RUN: %clangxx -fsycl -fsycl-device-lib=libm-fp64 %s -o %t.out
 // RUN: env SYCL_DEVICE_TYPE=HOST %t.out
 // RUN: %CPU_RUN_PLACEHOLDER %t.out
-// RUN: %ACC_RUN_PLACEHOLDER %t.out
```

Comment:
Can you please explain why you had to remove this?

Reply:
Hi, @vzakhari 
As we haven;t required "accelerator" in test, I didn't enable "ACC".
If we enabled ACC, do we need to require "accelerator" in the test or just requiring cpu is enough for both ACC and CPU?

Reply:
Thank you for explaining.  If there is no `ACC` device, then the `ACC` placeholder will always evaluate to `true` and will not run the test executable.  I will mark particular lines that we do not need to modify in this PR.